### ISSUE TITLE
OKX watch_my_trades does not work

### ISSUE THREAD

**Issue by WillPCowan** [ccxt/ccxt#19759]
### Operating System

MacOS

### Programming Languages

Python

### CCXT Version

4.1.8

### Description

It appears that CCXT Pro's OKX connector does not return any trades via `watch_my_trades`. The trade updates can be extracted from watch_orders, by accessing the original okx message and identifying fill updates. Given the script below, running it will never yield a trade message (in my experience). 

It looks like `watch_my_trades` is implemented for [okx ](https://github.com/ccxt/ccxt/blob/master/python/ccxt/pro/okx.py) so I guess this is a bug?

Verbose logs are attached here: [output.log](https://github.com/ccxt/ccxt/files/13217691/output.log)

### Code

```
import asyncio
import ccxt.pro
from ccxt.async_support.base.exchange import Exchange

API_KEY = "***"
SECRET = "***"
PASSWORD = "***"
INSTRUMENT_ID = "BTC/USDC"

async def main_async():

    exchange = ccxt.pro.okx({
        "apiKey": API_KEY,
        "secret": SECRET,
        "password": PASSWORD,
    })
    exchange.verbose = True
    
    async def stream(exchange: Exchange) -> None:
        async def stream_orders() -> None:
            print("Starting to stream orders")
            try:
                while True:
                    private_orders = await exchange.watch_orders(
                        INSTRUMENT_ID
                    )
                    for ccxt_order in private_orders:
                        print(f"[ORDER] {ccxt_order}")
                        if ccxt_order["info"]["state"] in ("partially_filled", "filled"):
                            print("There was a fill, there should be a [TRADE] message.")
            except Exception as e:
                print("Error streaming orders")

        async def stream_trades() -> None:
            print("Starting to stream trades")
            try:
                while True:
                    private_trades = await exchange.watch_my_trades(
                        INSTRUMENT_ID
                    )
                    for ccxt_trade in private_trades:
                        print(f"[TRADE] {ccxt_trade}")
            except Exception as e:
                print("Error streaming trades")
        
        await asyncio.gather(
            stream_orders(),
            stream_trades(),
        )
        
    async def place_fillable_order(exchange: Exchange) -> None:
        await asyncio.sleep(4)
        
        # Get midprice
        print("Getting midprice")
        order_book = await exchange.fetch_order_book(INSTRUMENT_ID)
        midprice = (order_book["asks"][0][0] + order_book["bids"][0][0]) / 2
        print(f"Midprice {midprice}")
        
        fillable_price = int(midprice * 1.1)
        print(f"Fillable price {fillable_price}")
        
        print("Placing order")
        try:
            order = await exchange.create_limit_buy_order(
                symbol=INSTRUMENT_ID,
                amount=0.00001,
                price=fillable_price,
            )
            print(f"[ORDER] {order}")
        except Exception as e:
            print("Error placing order")
            print(e)
            return
    
    try:
        await asyncio.gather(
            stream(exchange),
            place_fillable_order(exchange),
        )
    finally:
        await exchange.close()


def main():
    try:
        asyncio.run(main_async())
    except Exception as e:
        print("Failed to exit gracefully")


if __name__ == "__main__":
    main()
```

### Custom Logs
```
Starting to stream orders
Starting to stream trades
Getting midprice
Midprice 34292.25
Fillable price 37721
Placing order
[ORDER] {'info': {'clOrdId': 'e847386590ce4dBCca7e58a202bebe66', 'ordId': '639590929154109442', 'sCode': '0', 'sMsg': 'Order placed', 'tag': 'e847386590ce4dBC'}, 'id': '639590929154109442', 'clientOrderId': '***', 'timestamp': None, 'datetime': None, 'lastTradeTimestamp': None, 'lastUpdateTimestamp': None, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': None, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': None, 'cost': None, 'amount': None, 'filled': None, 'remaining': None, 'status': None, 'fee': None, 'trades': [], 'reduceOnly': False, 'fees': []}
[ORDER] {'info': {'accFillSz': '0', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '0', 'cTime': '1698762360535', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': 'e847386590ce4dBCca7e58a202bebe66', 'code': '0', 'execType': '', 'fee': '0', 'feeCcy': 'BTC', 'fillFee': '0', 'fillFeeCcy': '', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '', 'fillPnl': '0', 'fillPx': '', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0', 'fillTime': '', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '34288.7', 'lever': '0', 'msg': '', 'notionalUsd': '0.37721000000000005', 'ordId': '639590929154109442', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '37721', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'live', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': 'e847386590ce4dBC', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '', 'uTime': '1698762360535'}, 'id': '639590929154109442', 'clientOrderId': '***', 'timestamp': 1698762360535, 'datetime': '2023-10-31T14:26:00.535Z', 'lastTradeTimestamp': None, 'lastUpdateTimestamp': 1698762360535, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 37721.0, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': None, 'cost': 0.0, 'amount': 1e-05, 'filled': 0.0, 'remaining': 1e-05, 'status': 'open', 'fee': {'cost': 0.0, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 0.0, 'currency': 'BTC'}]}
[ORDER] {'info': {'accFillSz': '0.00001', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '34292.3', 'cTime': '1698762360535', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': 'e847386590ce4dBCca7e58a202bebe66', 'code': '0', 'execType': 'T', 'fee': '-0.0000000028', 'feeCcy': 'BTC', 'fillFee': '-0.0000000028', 'fillFeeCcy': 'BTC', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '0.342923', 'fillPnl': '0', 'fillPx': '34292.3', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0.00001', 'fillTime': '1698762360536', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '34288.7', 'lever': '0', 'msg': '', 'notionalUsd': '0.37721000000000005', 'ordId': '639590929154109442', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '37721', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'filled', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': 'e847386590ce4dBC', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '14035481', 'uTime': '1698762360538'}, 'id': '639590929154109442', 'clientOrderId': '***', 'timestamp': 1698762360535, 'datetime': '2023-10-31T14:26:00.535Z', 'lastTradeTimestamp': 1698762360536, 'lastUpdateTimestamp': 1698762360538, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 37721.0, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': 34292.3, 'cost': 0.342923, 'amount': 1e-05, 'filled': 1e-05, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 2.8e-09, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 2.8e-09, 'currency': 'BTC'}]}
There was a fill, there should be a [TRADE] message.
```

**Comment by sc0Vu** [comment]
@WillPCowan

I can't reproduce this from my side. Would you like to paste verbose log for the connection?

**Comment by WillPCowan** [comment]
Hey @sc0Vu thanks for checking this out for me, verbose logs are attached as a file (large), see: "Verbose logs are attached here: [output.log](https://github.com/ccxt/ccxt/files/13217691/output.log)".

So you are getting the trade message, what version of CCXT are you using. Any ideas on why I might not be? I can use the same script with different exchanges and successfully get trade messages, just not for OKX.

**Comment by WillPCowan** [comment]
Cheers guys ❤️

**Comment by WillPCowan** [comment]
Assuming this fix is in PyPi 4.1.35 released on Nov 1, I don't think the issue is fixed but now more broken 😆 

There are still no trade messages, but I'm getting duplicate order messages now (for some reason it is always 2 open order and 4 closed order). PyPi 4.1.34 does not have this problem.

Verbose logs:
[output.log](https://github.com/ccxt/ccxt/files/13235961/output.log)


```
[2023-11-02 07:54:27,340] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '0', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': '', 'fee': '0', 'feeCcy': 'BTC', 'fillFee': '0', 'fillFeeCcy': '', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '', 'fillPnl': '0', 'fillPx': '', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0', 'fillTime': '', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'live', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '', 'uTime': '1698911667201'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': None, 'lastUpdateTimestamp': 1698911667201, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': None, 'cost': 0.0, 'amount': 1e-05, 'filled': 0.0, 'remaining': 1e-05, 'status': 'open', 'fee': {'cost': 0.0, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 0.0, 'currency': 'BTC'}]}
[2023-11-02 07:54:27,341] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '0', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': '', 'fee': '0', 'feeCcy': 'BTC', 'fillFee': '0', 'fillFeeCcy': '', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '', 'fillPnl': '0', 'fillPx': '', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0', 'fillTime': '', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'live', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '', 'uTime': '1698911667201'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': None, 'lastUpdateTimestamp': 1698911667201, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': None, 'cost': 0.0, 'amount': 1e-05, 'filled': 0.0, 'remaining': 1e-05, 'status': 'open', 'fee': {'cost': 0.0, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 0.0, 'currency': 'BTC'}]}
[2023-11-02 07:54:27,342] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0.00001', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '35260.8', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': 'T', 'fee': '-0.0000000028', 'feeCcy': 'BTC', 'fillFee': '-0.0000000028', 'fillFeeCcy': 'BTC', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '0.352608', 'fillPnl': '0', 'fillPx': '35260.8', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0.00001', 'fillTime': '1698911667201', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'filled', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '14105184', 'uTime': '1698911667202'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': 1698911667201, 'lastUpdateTimestamp': 1698911667202, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': 35260.8, 'cost': 0.352608, 'amount': 1e-05, 'filled': 1e-05, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 2.8e-09, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 2.8e-09, 'currency': 'BTC'}]}
[2023-11-02 07:54:27,342] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0.00001', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '35260.8', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': 'T', 'fee': '-0.0000000028', 'feeCcy': 'BTC', 'fillFee': '-0.0000000028', 'fillFeeCcy': 'BTC', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '0.352608', 'fillPnl': '0', 'fillPx': '35260.8', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0.00001', 'fillTime': '1698911667201', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'filled', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '14105184', 'uTime': '1698911667202'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': 1698911667201, 'lastUpdateTimestamp': 1698911667202, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': 35260.8, 'cost': 0.352608, 'amount': 1e-05, 'filled': 1e-05, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 2.8e-09, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 2.8e-09, 'currency': 'BTC'}]}
[2023-11-02 07:54:27,343] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0.00001', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '35260.8', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': 'T', 'fee': '-0.0000000028', 'feeCcy': 'BTC', 'fillFee': '-0.0000000028', 'fillFeeCcy': 'BTC', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '0.352608', 'fillPnl': '0', 'fillPx': '35260.8', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0.00001', 'fillTime': '1698911667201', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'filled', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '14105184', 'uTime': '1698911667202'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': 1698911667201, 'lastUpdateTimestamp': 1698911667202, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': 35260.8, 'cost': 0.352608, 'amount': 1e-05, 'filled': 1e-05, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 2.8e-09, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 2.8e-09, 'currency': 'BTC'}]}
[2023-11-02 07:54:27,343] [INFO] [MainThread] - [ORDER] {'info': {'accFillSz': '0.00001', 'algoClOrdId': '', 'algoId': '', 'amendResult': '', 'amendSource': '', 'attachAlgoClOrdId': '', 'avgPx': '35260.8', 'cTime': '1698911667201', 'cancelSource': '', 'category': 'normal', 'ccy': '', 'clOrdId': '80383920821c4da59678b27d0835a24d', 'code': '0', 'execType': 'T', 'fee': '-0.0000000028', 'feeCcy': 'BTC', 'fillFee': '-0.0000000028', 'fillFeeCcy': 'BTC', 'fillFwdPx': '', 'fillMarkPx': '', 'fillMarkVol': '', 'fillNotionalUsd': '0.352608', 'fillPnl': '0', 'fillPx': '35260.8', 'fillPxUsd': '', 'fillPxVol': '', 'fillSz': '0.00001', 'fillTime': '1698911667201', 'instId': 'BTC-USDC', 'instType': 'SPOT', 'lastPx': '35260.8', 'lever': '0', 'msg': '', 'notionalUsd': '0.35263900000000004', 'ordId': '640217166700539907', 'ordType': 'limit', 'pnl': '0', 'posSide': '', 'px': '35263.9', 'pxType': '', 'pxUsd': '', 'pxVol': '', 'quickMgnType': '', 'rebate': '0', 'rebateCcy': 'USDC', 'reduceOnly': 'false', 'reqId': '', 'side': 'buy', 'slOrdPx': '', 'slTriggerPx': '', 'slTriggerPxType': '', 'source': '', 'state': 'filled', 'stpId': '', 'stpMode': '', 'sz': '0.00001', 'tag': '', 'tdMode': 'cash', 'tgtCcy': '', 'tpOrdPx': '', 'tpTriggerPx': '', 'tpTriggerPxType': '', 'tradeId': '14105184', 'uTime': '1698911667202'}, 'id': '640217166700539907', 'clientOrderId': '80383920821c4da59678b27d0835a24d', 'timestamp': 1698911667201, 'datetime': '2023-11-02T07:54:27.201Z', 'lastTradeTimestamp': 1698911667201, 'lastUpdateTimestamp': 1698911667202, 'symbol': 'BTC/USDC', 'type': 'limit', 'timeInForce': None, 'postOnly': None, 'side': 'buy', 'price': 35263.9, 'stopLossPrice': None, 'takeProfitPrice': None, 'stopPrice': None, 'triggerPrice': None, 'average': 35260.8, 'cost': 0.352608, 'amount': 1e-05, 'filled': 1e-05, 'remaining': 0.0, 'status': 'closed', 'fee': {'cost': 2.8e-09, 'currency': 'BTC'}, 'trades': [], 'reduceOnly': False, 'fees': [{'cost': 2.8e-09, 'currency': 'BTC'}]}
```

**Comment by KhushbirD** [comment]
Hi there, Is there any resolution for the above problem? I see the duplicates in each watch_orders() and watch_my_trades() as well as soon as i watch for both.
